src/NzbDrone.Core/ImportLists/Simkl/SimklParser.cs

```diff
@@ -55,7 +55,7 @@ public virtual IList<ImportListItemInfo> ParseResponse(ImportListResponse import
                     }
                     else
                     {
-                        Logger.Warn("Skipping info grabbing for '{0}' because it is a movie or it is not the first season of the show", show.Show.Title);
+                        Logger.Warn($"Skipping info grabbing for '{show.Show.Title}' because it is missing a TVDB ID, or it is movie or music video");
```

Comment:
Similar here, but instead of movie it should say something like "unsupported content type"